### ISSUE TITLE
Crash on Install wallpaper

### ISSUE THREAD

**Issue by shuvashish76** [zadeviggers/wallpaper#3]
**Steps to produce:**
Open tge app & simply click on Install walpaper

**Logs:**
```
FATAL EXCEPTION: main
Process: net.viggers.zade.wallpaper, PID: 30280
java.lang.NumberFormatException: For input string: "2.13136179E9"
	at java.lang.Integer.parseInt(Integer.java:608)
	at java.lang.Integer.valueOf(Integer.java:794)
	at r2.g.g(Unknown Source:218)
	at r2.g.<init>(Unknown Source:196)
	at net.viggers.zade.wallpaper.WallpaperService.onCreateEngine(Unknown Source:2)
	at android.service.wallpaper.WallpaperService$IWallpaperEngineWrapper.executeMessage(WallpaperService.java:1265)
	at com.android.internal.os.HandlerCaller$MyHandler.handleMessage(HandlerCaller.java:37)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6518)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```
**Device details:**
 - Android: 8.1.0
 - Model: Lenovo Tab4 8 plus
 - App version: 15.7 (22)

**Comment by zadeviggers** [comment]
Did you install via F-Droid or direct APK?

**Comment by zadeviggers** [comment]
Can reproduce in simulator.

**Comment by shuvashish76** [comment]
> Did you install via F-Droid or direct APK?

F-Droid yes.

**Comment by zadeviggers** [comment]
This should be fixed in the next 15.8 release
